ansible/roles/dom0-defaults/tasks/main.yml

```diff
@@ -0,0 +1,18 @@
+---
+- name: install some common packages
+  apt:
+    name: "{{ item }}"
+    state: latest
+    update_cache: yes
+    cache_valid_time: 28800
+    install_recommends: false
+  with_items:
+    - ca-certificates
+    - tmux
```

Comment:
It's mostly the same with ^B default instead of ^A (so readline's shortcut go-to-beginning-of-the-line works) and some nice features like tiled layout of split-screen and sending keyboard input to every pane (that's nice to feature to run same interactive command on 9..12 servers).
I've added screen, but I'd suggest to give `tmux` a try :)